include/openssl/ssl.h

```diff
@@ -1899,6 +1899,15 @@ int DTLSv1_listen(SSL *s, BIO_ADDR *client);
 # ifndef OPENSSL_NO_CT
 
 /*
+ * A callback for verifying that the received SCTs are sufficient.
+ * Expected to return 1 if they are sufficient, otherwise 0.
+ * May return a negative integer if an error occurs.
```

Comment:
Why do we allow negative integers? Unless the library code does something meaningful with them, I think we should require 1/0.

"A connection should be aborted" -> "the connection will be aborted"?